drivers/hmis_supplemental/app/models/hmis_supplemental/field_value.rb

```diff
@@ -14,7 +16,6 @@ module HmisSupplemental
   class FieldValue < GrdaWarehouseBase
     self.table_name = 'hmis_supplemental_field_values'
     belongs_to :data_set, class_name: 'HmisSupplemental::DataSet'
-    serialize :data, type: Hash
```

Comment:
Is this removal intentional?  Maybe the column is jsonb and this is the default?

Also, it appears there are duplicate copyright notices above.

Reply:
Yes, sorry I should have called that out. This pairs with the migration in db/warehouse/migrate/20250415143753_fix_json_serialization.rb

This was mistakenly storing the serialized value as a string inside of a JSON field rather than as JSON (as intended)